README.md

```diff
@@ -85,14 +85,14 @@ export NODEREPORT_VERBOSE=yes|no
 
 ## Examples
 
-To see examples of NodeReports generated from these events you can run the
-demonstration applications provided in the nodereport github repository. These are
+To see examples of reports generated from these events you can run the
+demonstration applications provided in the node-report github repository. These are
```

Comment:
most projects put user-focussed info at top of readme, and other info at bottom (and ignore that its not really intended for the npmjs.org audience). I rarely read the info on npmjs.org past the first screen, I usually just jump directly to github even to read using the link at the top-right. You could also rip the dev-focussed stuff into a CONTRIBUTING.md in the top-level, which would get it off npmjs.org. And then link to it from the README, or not? I wouldn't put dev info in a wiki, its too far from the source.